SEBrowser/Scripts/TSX/Components/TrendData/TrendPlot/LineGraph.tsx

```diff
@@ -143,10 +121,11 @@ const LineGraph = React.memo((props: IProps) => {
     }, [props.PlotFilter]);
 
     React.useEffect(() => {
-        const centerTime: moment.Moment = moment.utc(props.TimeFilter.date + props.TimeFilter.time, timeFilterFormat);
-        const startTime: number = centerTime.add(-props.TimeFilter.windowSize, formatWindowUnit(props.TimeFilter.timeWindowUnits)).valueOf();
-        // Need to move back in the other direction, so entire window
-        const endTime: number = centerTime.add(2 * props.TimeFilter.windowSize, formatWindowUnit(props.TimeFilter.timeWindowUnits)).valueOf();
+        const startMoment: moment.Moment = moment.utc(props.TimeFilter.start, dateTimeFormat);
+        const endMoment: moment.Moment = moment.utc(props.TimeFilter.end, dateTimeFormat);
+
+        const startTime: number = startMoment.valueOf();
+        const endTime: number = endMoment.valueOf();
```

Comment:
```suggestion
        const startTime: number = moment.utc(props.TimeFilter.start, dateTimeFormat).valueOf();
        const endTime: number = moment.utc(props.TimeFilter.end, dateTimeFormat).valueOf();
```
No need to have this assignment, it makes it a little more hard to read in my opinion.